server/base/src/main/java/org/apache/accumulo/server/init/Initialize.java

```diff
@@ -504,7 +503,7 @@ private void initFileSystem(SiteConfiguration siteConfig, Configuration hadoopCo
     // initialize initial system tables config in zookeeper
     initSystemTablesConfig(zoo, Constants.ZROOT + "/" + uuid, hadoopConf);
 
-    VolumeChooserEnvironment chooserEnv = new VolumeChooserEnvironmentImpl(ChooserScope.INIT, null);
+    VolumeChooserEnvironment chooserEnv = new InitVolumeChooserEnvironmentImpl(siteConfig);
```

Comment:
If passing keyextents here, would need to create three keyextents and pass those to three separate InitVolumeChooserEnvironmentImpl.  The key extents would be something like the following.

```java
// move creation of splitPoint here.
var tableMetadataExtent = new KeyExtent(MetadataTable.ID, splitPoint, null);
var defaultMetadataExtent = new KeyExtent(MetadataTable.ID, null, splitPoint);
var replicationExtent = new KeyExtent(ReplicationTable.ID, null, null);
```

If doing this, I think it would make sense to refactor the internal Tablet object to take a KeyExtent.